src/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:lts-alpine as ui-frontend-builder
+
+COPY frontend /srv/frontend
+WORKDIR /srv/frontend
+RUN npm ci \
+ && npm run build:prod
+
+FROM python:alpine3.18 as s3gw-ui
+LABEL Name=s3gw-ui
+
+COPY --from=ui-frontend-builder /srv/frontend/dist/s3gw-ui /srv/frontend/dist/s3gw-ui
+COPY backend /srv/backend
+COPY requirements.txt /srv
+COPY s3gw_ui_backend.py /srv
+WORKDIR /srv
+RUN pip install --no-cache-dir -r requirements.txt
+STOPSIGNAL SIGINT
+EXPOSE 8080
+ENTRYPOINT [ "python", "./s3gw_ui_backend.py" ]
```

Comment:
sure we know: that is implicit with the base image version: `FROM python:alpine3.18 as s3gw-ui`

Reply:
Let me make it a bit clear, here there is python ./s3gw_ui_backend.py, which in general can be python2, or so... However in the other Makefile, there is python3:
```
run-ui-backend:
 	cd src \
 	&& python3 -m venv venv \
 	&& source venv/bin/activate \
 	&& S3GW_DEBUG=1 python3 ./s3gw_ui_backend.py
```
Which can make a confusion.

Reply:
ah, yes I see what you mean; well, yes this could be a bit confusing; but the fact that in this specific Dockerfile context `python` means actually `python3` is an immediate fact, because it derives from the usage of a specialized docker image: `python:alpine3.18`.
I think that trying to fix this to uniform to other scripts could be more confusing and convoluted than leaving it as it is now.
Consider that the Makefile is thought for working on the host of the developer, but here we are defining the real environment used for the production.

Reply:
If you fine with that I would rather approve it and let it merge.

Reply:
I'm fine with that!